src/google_apps_clj/google_sheets_v4.clj

```diff
@@ -104,31 +161,16 @@
               (-> (NumberFormat.)
                   (.setType "CURRENCY"))))))))
 
-(defn date-time
-  "using the given excel date formatting string"
-  [date-time format-patterm]
-  (-> (CellData.)
-      (.setUserEnteredValue
-       (-> (ExtendedValue.)
-           ;; Heaven help us. Excel serial time and casting to a float.
-           (.setNumberValue
-            (double (+ 2 (time/in-days (time/interval (time/date-time 1900 1 1)
-                                                      date-time)))))))
-      (.setUserEnteredFormat
-       (-> (CellFormat.)
-           (.setNumberFormat
-            (-> (NumberFormat.)
-                (.setType "DATE")
-                (.setPattern format-patterm)))))))
-
 (defn row->row-data
   "google-ifies a row (list of columns) of type string?, number? keyword? or CellData."
   [row]
   (-> (RowData.)
       (.setValues (map coerce-to-cell-data row))))
 
 (defn write-sheet
-  "writes values to a specific sheet (tab). Breaks down requests into batches of ~10k cells."
+  "writes values to a specific sheet (tab). Breaks down requests into batches of ~10k cells.
+  Overwrites all the existing rows on the sheet.  Doesn't alter the number of
+  columns on the sheet and so writing more columns than the sheet has will error"
```

Comment:
Yep that is a good point. I had thought about that and started to wonder if that would bring us to the point that one where clients should just use the google Java API directly since it would be such a close match to what's already provided. I.e. I started going down that road and felt myself writing the classic 1:1 wrap-a-library-with-another-layer-of-library. But I'm not sure if I can fully articulate the value prop of this library and what functionality belongs in and what belongs out, which is a bad sign.

In any case I don't think allowing batching commands in this API is a bad idea and would be happy going down that road.